mlir/utils/jenkins/static-checks/premerge-checks.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
```

Comment:
The formatting of this file is wired. You have the tabspace to be 2/3/4. The python files under llvm folder are all 2 space by default. I suggest you to run static checks on this file yourself to fix the formatting issues.

Reply:
I'm referencing to the formatting of [this file](https://github.com/ROCmSoftwarePlatform/llvm-project-mlir/blob/miopen-dialect/mlir/utils/generate-test-checks.py). I believe `autopep8` can do it for you.